### PR TITLE
add new file en-US/snippets/2017/mar2017.lang

### PR DESCRIPTION
**Pull request by peiying2**
per bug https://bugzilla.mozilla.org/show_bug.cgi?id=1336204.

part two from the bug will come after some clarification.

### CODE REVIEW

File: en-US/snippets/2017/mar2017.lang [hunk 1]
```diff
@@ -0,0 +1,87 @@
+## NOTE: reference bug 1336204; Deadline: 24 February, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Link for Google Play: https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp
+## NOTE: Link for App Store: https://itunes.apple.com/app/apple-store/id989804926?mt=8
+## NOTE: Campaign: Mobile Promotions
+
+
+## MAX_LENGTH: 120 characters
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017.lang]: I confess I'm quite confused by these max-length. In the bug you wrote that `Under Mobile Promotions section, the headers have character limitation of 120.`. Can you confirm the limitation is only for the first line of the snippet? 120+180 seems like a huge size, especially when English is a mere 33 characters.

Even with that clarified, it's unclear why only this one has the 120 characters limit set.

One more thing: when you add a limit with `## MAX_LENGTH`, you also need to add a comment for localizers to tell them that. That's valid for the entire file.

Comment (peiying2) [inline en-US/snippets/2017/mar2017.lang]: I can remove this one (120) consider the current length. When I asked about the character limitation is applied to all or just the longer ones.  He said the short ones are "headers" and should be kept at 120, so the two lines are two separate entries.  I only noted in the longer ones.  But even these are so short, that they can't easily reach the limit.  I can skip the short ones and only note the long (full text) to the limit of 180.

Comment (flodolo) [inline en-US/snippets/2017/mar2017.lang]: Just to be safe I would put the limit on all of them.

File: en-US/snippets/2017/mar2017.lang [hunk 2]
```diff
@@ -0,0 +1,90 @@
+## NOTE: reference bug 1336204; Deadline: 24 February, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Link for Google Play: https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp
+## NOTE: Link for App Store: https://itunes.apple.com/app/apple-store/id989804926?mt=8
+## NOTE: Campaign: Desktop Snippets
+
+
+## MAX_LENGTH: 120 characters
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017.lang]: As explained in the previous comment, every time you add a metatag to impose a maximum length, you **need** to add a localization comment explaining it. In localized files you'll only see this comment, not the meta tag.

In doubt, look at the previous snippet files you created, e.g. https://github.com/mozilla-l10n/engagement-l10n/blob/master/en-US/snippets/2017/feb2017.lang#L14-L19

File: en-US/snippets/2017/mar2017_b.lang [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+Unlike your attic, Firefox is easy to clean. <a>Just click the Forget Button and presto!</a> No more browser history.
+
+
+## MAX_LENGTH: 180 characters
+# Link: https://addons.mozilla.org/firefox/themes
+;Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
+Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
+
+
+# Appears on Mobile Activation screen; Refer to the screenshot.
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: Which screenshot? Question applies to all comments similar to this one

File: en-US/snippets/2017/mar2017_b.lang [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Send
+
+
+# Appears on Mobile Activation screen; Refer to the screenshot.
+;Enter your email
+Enter your email
+
+
+# Appears on Mobile Activation screen; Refer to the screenshot.
+;App store
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: Apple's trademark are pretty clear that the right spelling is "App Store"
https://developer.apple.com/app-store/marketing/guidelines/

Can it be translated/transliterated? Probably not

File: en-US/snippets/2017/mar2017_b.lang [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+
+## MAX_LENGTH: 180 characters
+# Link: https://addons.mozilla.org/firefox/themes
+;Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
+Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
+
+
+# Appears on Mobile Activation screen; Refer to the screenshot.
+;Send
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: This sounds like a button, and buttons usually have a maximum length. I'm not worried about Chinese, more about Indonesian.

File: en-US/snippets/2017/mar2017_b.lang [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+Send
+
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: What happened to "Remove this" from the bug?

File: en-US/snippets/2017/mar2017_b.lang [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+Enter your email
+
+
+# Appears on Mobile Activation screen; Refer to the screenshot.
+;App store
+App store
+
+
+# Appears on Mobile Activation screen.
+;Google play
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: Same here: "Google Play"

File: en-US/snippets/2017/mar2017_b.lang (hunk 6 above)
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: Uhm, commented in the bug.

File: en-US/snippets/2017/mar2017_b.lang [hunk 8]
```diff
@@ -6,115 +6,148 @@
...
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 # Link: https://addons.mozilla.org/firefox/themes
 ;Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
 Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
 
 
-# Appears on Mobile Activation screen; Refer to the screenshot.
+# Appears on Mobile Activation screen; For context, refer to https://bug1336204.bmoattachments.org/attachment.cgi?id=8833047
+# This is a button. Be mindful of the character limit.
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: I would say "keep the translation as short as possible", given that we don't tell them what the limit is.

File: en-US/snippets/2017/mar2017_b.lang [hunk 9]
```diff
@@ -115,26 +115,27 @@ Your browser is ready for a makeover. Don’t worry, you’ve got tons of option
...
 
 # Appears on Mobile Activation screen; For context, refer to https://bug1336204.bmoattachments.org/attachment.cgi?id=8833047
-# This is a button. Be mindful of the character limit.
+# This is a button. Keep the translation short.
 ;Send
 Send
 
 
+# This is a tool tip.
+;Remove this
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: Where's the "translation" of this string?

File: en-US/snippets/2017/mar2017_b.lang [hunk 10]
```diff
@@ -115,26 +115,27 @@ Your browser is ready for a makeover. Don’t worry, you’ve got tons of option
 
 
 # Appears on Mobile Activation screen; For context, refer to https://bug1336204.bmoattachments.org/attachment.cgi?id=8833047
-# This is a button. Be mindful of the character limit.
+# This is a button. Keep the translation short.
 ;Send
 Send
 
 
+# This is a tool tip.
```
Comment (flodolo) [inline en-US/snippets/2017/mar2017_b.lang]: nit: tooltip, maybe also "This is a tooltip on the UI element to dismiss the widget"?

**Pull request merged**